src/extensions/queue-repeat/queue-repeat.js

```
@@ -0,0 +1,409 @@
+                seen.add(uri);
+                repeatList.push(uri);
+            }
+        }
+
+        previousTrackUri = currentUri;
+        isActive         = true;
+
+        updateButtonVisual(true);
+
+        log(`Queue Repeat on. ${repeatList.length} track(s) in repeat list.`);
+        repeatList.forEach((uri, i) => log(`  [${i + 1}] ${uri}`));
```

Biome (2.4.13)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)